interface/wx/event.h

```diff
@@ -1403,6 +1403,9 @@ enum wxKeyCategoryFlags
     to 31.
     Since version 2.9.2, the enum values @c WXK_CONTROL_A - @c WXK_CONTROL_Z
     can be used instead of the non-descriptive constant values 1-26.
+    Since version 3.3.0, if non-US/English keyboard layout is selected,
+    Unicode field of char events contains original Unicode character
+    as it would be without Control.
```

Comment:
I think it would be more useful to just document the current behaviour and just say that `GetUnicodeKey()` returns the key code that would be generated by the key press without `Control`, and that the adjustment described here is only done for `GetKeyCode()`.

Reply:
Updated